packages/desktop/src/renderer/components/ActiveSketch/ActiveSketch.tsx

```diff
@@ -18,6 +23,14 @@ export const ActiveSketch = () => {
 
   const selectedParam = useSelectedParam()
 
+  const pluginView = engine.plugins.map((plugin) => (
```

Comment:
Grabbing values directly from the engine object makes me nervous here, because React has no way of knowing if that object has changed and won't update accordingly. It's fine for now because I think in practice the plugins will always load before this component is mounted and then never change, just worth noting that we'll probably want to clean this up at some point.